SiteAssets/pages/3.0/scripts/configData.js

```diff
@@ -20,5 +19,34 @@ configData = {
     'Construction manager approval comment populates cdmSMMitigationSuggestion': false, // Do you want the approval comment from the construction manager's review to populate the site manager's mitigation suggestion
     'Full admin edit rights': false, // Do you want the admin to be able to edit any field, at any point in the workflow?
     'Include contract': true, // Do you want to have the option to assign contracts to hazards?
-    'Exportable workflow states': ['Requires mitigation', 'Assessment in progress', 'Under peer review', 'Under design manager review']
+    'Exportable workflow states': ['Requires mitigation', 'Assessment in progress', 'Under peer review', 'Under design manager review'],
+    
+    //Editable workflow config section. Patrick Hsu, 29 Jan 2024
+    'Workflow' : { 
+        'initiatereview' : { 
+            'nextWorkFlowState': 'Under peer review', 
+            'userRoles': ['Designer','Construction Engineer','Design Manager','Construction Manager','Principal Designer'] // who can review/edit the hazard at this point
+        },
+        'peerreview' : { 
+            'nextWorkFlowState1': 'Under Construction Manager review', 
```

Comment:
I think there should only be one nextWorkFlowState here and it should be 'Under design manager review'